lib/external_count_fetcher.rb

```diff
@@ -6,61 +6,79 @@
 #   # => {}
 #
 #   ExternalCountFecther.new.counts
-#   # => {supporter_count: 12, volunteer_count: 12, called_voters_count: 12, called_reps_count: 12, house_supporters: 12, senate_supporters: 12, donations_total: 12, donations_count: 12}
+#   # => {supporter_count: 12, volunteer_count: 12, called_voters_count: 12, reps_calls_count: 12, house_supporters: 12, senate_supporters: 12, donations_total: 12, donations_count: 12}
```

Comment:
Line is too long. [185/80]<br>Incorrect indentation detected (column 0 instead of 1).